Stack/Opc.Ua.Core/Types/Encoders/JsonEncoder.cs

```diff
@@ -1241,17 +1261,11 @@ public void WriteExtensionObject(string fieldName, ExtensionObject value)
                 return;
             }
 
+            // reversible encoding
             PushStructure(fieldName);
 
-            if (UseReversibleEncoding)
-            {
-                var nodeId = ExpandedNodeId.ToNodeId(value.TypeId, Context.NamespaceUris);
-                WriteNodeId("TypeId", nodeId);
-            }
-            else
-            {
-                WriteExpandedNodeId("TypeId", value.TypeId);
-            }
```

Comment:
does non reversible really write the urihere?

Reply:
reverted for now. I need to further clarify this.